edge/pkg/common/message/message.go

```diff
@@ -7,12 +7,13 @@ import (
 //constant defining node connection types
 const (
 	ResourceTypeNodeConnection = "node/connection"
-	OperationNodeConnection    = "publish"
+	
+	OperationNodeConnection = "publish"
 	SourceNodeConnection       = "edgehub"
 )
 
 //BuildMsg returns message object with router and content details
-func BuildMsg(group, parentID, sourceName, resource, operation string, content interface{}) *model.Message {
+func BuildMsg(group, parentId, sourceName, resource, operation string, content interface{}) *model.Message {
```

Comment:
Golint naming: func parameter parentId should be parentID. [More info](https://golang.org/wiki/CodeReviewComments#initialisms). <!-- golint -->